### DISCUSSION TITLE
Add Component and scroll to it.

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3870] [Q&A]
Hello @artf,
  How to scroll to newly added component ?
  ```js
      cmp.addComponent(data.results, {
          at: 1
      })
  ```
   After adding new component how to scroll to it ?

**Comment by artf** [comment]
You can use [`editor.Canvas.scrollTo`](https://grapesjs.com/docs/api/canvas.html#scrollto)

**Reply by iabhiyaan** [reply]
How can we do for newly added component ?

**Reply by artf** [reply]
cmp.addComponent returns added components

**Reply by iabhiyaan** [reply]
Ok got it! Thankyou for your response
